.github/workflows/ack.yml

```diff
@@ -37,10 +37,13 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         continue-on-error: true
 
+      - name: Dump the context
+        uses: crazy-max/ghaction-dump-context@v2
```

Comment:
Is this supposed to be here in the final PR? It looks useful, I'm just not sure what it's for running all the time.